extensions/aerial-photo/examples/item.json

```diff
@@ -1,11 +1,11 @@
 {
   "stac_version": "1.0.0",
-  "stac_extensions": ["https://linz.github.io/stac/_STAC_VERSION_/aerial-photo/schema.json"],
+  "stac_extensions": ["https://stac.linz.govt.nz/_STAC_VERSION_/aerial-photo/schema.json"],
   "type": "Feature",
   "id": "72360",
   "geometry": null,
   "properties": {
-    "datetime": "1952-04-23T00:00:00.000Z",
+    "datetime": "1952-04-22T12:00:00Z",
```

Comment:
Should the datetime format change be a separate PR? Also, why offset the hour? Looks like some unexpected NZ datetime change.

Reply:
The offset is because all dates should be in UTC but the source is in NZ time.

Reply:
Ah right. These changes all seem unrelated enough that they should ideally be a bunch of PRs. What do you think, @blacha?

Reply:
Sure, let me know how you would like me to split them up.

Reply:
I'd normally just create a new branch from origin/master for each independent change and commit/push them as a PR each. Basically, you might end up with one PR to change the URLs to stac.linz.govt.nz, another to not require eo:bands asset field in the Historical Imagery extension, etc. That way, even years from now someone could understand each change in isolation. It also helps when we sometimes have to revert a commit. If the commits are small and self-contained this is trivial, if they include a bunch of different changes it's really difficult to know which have to be reverted.

Reply:
Will do, thanks! Ignore this PR for now.